JobConfig/beam/TS-CRV.fcl

```diff
@@ -35,47 +35,7 @@ physics : {
    }
 
```

Comment:
I believe this file and many others in beam are no longer needed.

Reply:
This PR does not attempt to restructure JobConfig.  It only makes sure the existing JobConfig files are not broken by the code updates.

Reply:
That's fine for now.